shared/build.gradle.kts

```diff
@@ -434,4 +434,11 @@ mokkery {
     ignoreFinalMembers.set(true)
 }
 
-sentryKmp { autoInstall.commonMain.enabled = false }
+sentryKmp {
+    autoInstall.commonMain.enabled = false
+    if (autoInstall.cocoapods.sentryCocoaVersion.getOrElse("") == "~> 8.44.0") {
+        autoInstall.cocoapods.sentryCocoaVersion.set("~> 8.46.0")
+    } else {
+        throw IllegalStateException("sentry-kmp was updated, delete explicit sentry-cocoa version")
+    }
```

Comment:
If we never remove `.set("~> 8.46.0")`, we’ll never upgrade past 8.46.0 even once sentry-kmp suggests a higher version. I can’t make the code actually delete itself, but failing all the CI on that eventual dependabot PR should remind us to take a look